docs/analysis-workflows/rna-alignment.md

```diff
@@ -0,0 +1,35 @@
+---
+id: rna-alignment
+title: RNA-Seq Alignment
+sidebar_label: RNA-Seq Alignment
+platform_key: DOCS_RNA_PIPELINE
+---
+
+[ARGO RNA-Seq Alignment](https://github.com/icgc-argo-workflows/rna-seq-alignment/releases) workflow is a critical step in the RNA-Seq data analysis pipeline that involves mapping short RNA sequencing reads to a reference genome or transcriptome. The workflow accepts raw sequencing data in both FASTQ and BAM (aligned or unaligned) format, performs both STAR and HISAT2 alignments, and produces aligned reads in both genome and transcript coordinates, splice juntions and alignment QC reports.
```

Comment:
minor typo for "juntions" (should be "junctions")

Reply:
Thank you very much Hardeep, good eyes and nice catch.